### ISSUE TITLE
Make the scraping configurable

### ISSUE THREAD

**Issue by lisp-ceo** [rgarcia/dochub#39]
I'd ideally like to be able to swap-in and swap out components easily. If Python isn't relevant to me but a JS framework is, I'd like the ability to have dochub read my user dir for preferences and pull down any docs I need, relying on filename convention for namespacing and NPM for distribution.

E.g.

~/.dochub/config
~/.dochub/BackBone

npm install dochub-BackBone

More than happy to contribute this if you agree.

**Comment by brandonkm** [comment]
I really like the idea of this.  If I could swap out PHP and Python for the docs of whatever js framework I'm using, this could be even more of an excellent resource for client-side development.

**Comment by kevinburke** [comment]
This project hasn't been maintained or updated in more than a year - you should probably write the code you want and then  discuss ownership/deployment etc. with the authors of the project

**Comment by brandonkm** [comment]
Should have looked more closely at the dates and issues associated with this project.  Thanks for letting me know!

**Comment by rgarcia** [comment]
Yeah, sorry guys. I've mentioned this in other issues but me and and @vecter have both moved on to other projects, so we're looking for someone to take the reigns on dochub.
